OneSignalSDK/app/build.gradle

```diff
@@ -67,7 +67,7 @@ dependencies {
     implementation 'com.google.android.gms:play-services-location:12.0.0'
 
     // For Chrome tabs
-    compile 'com.android.support:customtabs:27.1.0'
+    api 'com.android.support:customtabs:27.1.0'
```

Comment:
api is the direct match for the old `compile` and it basically exposes the APAI from this library to any consumers of the module that depends on this plugin.. however, the consumer is the main app module and no one else is consuming it so, with that in mind, couldn't this just be `implementation` instead? 

🤔

Reply:
I was going to use `implementation` but another gradle file in this project uses `api` for this same dependency, so I kept it consistent.

https://github.com/OutSystems/OneSignal-Android-SDK/blob/7a7f954cb4e4aaafba0c7a08139976708deb7bc8/OneSignalSDK/onesignal/build.gradle#L43

On another note, that gradle uses the updated keywords unlike the changed file 🤔